packages/flutter/lib/src/widgets/scroll_activity.dart

```diff
@@ -260,7 +284,12 @@ class ScrollDragController implements Drag {
     // We negate the velocity here because if the touch is moving downwards,
     // the scroll has to move upwards. It's the same reason that update()
     // above negates the delta before applying it to the scroll offset.
-    delegate.goBallistic(-velocity);
+    velocity = -velocity;
```

Comment:
How about putting the minus sign on the line that first assigns velocity? I don't think that'd be any less understandable, but it would reduce the thrashing of the sign bit a little...

Right now when reading this function I get positively dizzy trying to keep track of what the sign means. :-)